edx_when/models.py

```diff
@@ -96,18 +106,23 @@ class ContentDate(models.Model):
     location = UsageKeyField(null=True, default=None, db_index=True, max_length=255)
     field = models.CharField(max_length=255, default='')
     active = models.BooleanField(default=True)
+    block_type = models.CharField(max_length=255, null=True)
+    course_version = models.ForeignKey(CourseVersion, null=True, on_delete=models.CASCADE)
 
     class Meta:
         """Django Metadata."""
 
         unique_together = ('policy', 'location', 'field')
+        indexes = [
```

Comment:
Nit: Add course_id and location here using the Index format instead of db_index on the field definition.

Reply:
Interesting - I'll try this and see how it changes the migration file.